Non-Interactive-Protocol.md

```diff
@@ -0,0 +1,173 @@
+congestion may fluctuate between fees negotiation and transaction propagation, party must adjust
+unilaterally feerate according to their confirmation preferences. A party may choose to wait
+for mempools draining before to adjust the feerate to avoid a fee waste as block height of 
+transition to execution phase doesn't matter, assuming it's before `contract_maturity_bound`.
+
+Reaching near `contract_maturiy_bound` requires to confirm the funding transaction quickly as it's
+now a security matter. After oracle signature release, a malicious party could try to double-spend
+an unconfirmed funding transaction to avoid loosing its collateral.
+
+
+# Execution Phase
+
+The execution phase transitions to the terminal phase either by a CET or a refund
+transaction confirmations.
+
+## Requirements
+
+A node:
+	- if an oracle signature is received:
+		- MUST finalize the CET with a local `SIGHASH_ALL` signature
+		- MUST broadcast the CET
+		- MAY spend its outcome output with a CPFP increasing package feerate to confirm soonly
+		- if the CET is not confirmed and `contract_timeout is 20 blocks ahead of local tip:
+			- MUST rebroadcast the CET
+			- MUST spend its outcome output with a CPFP increasing package feerate to confirm soonly
+			- MUST start a bumping timer of length `security_delay`
+		- if `security_delay` expires:
+			- MUST spend the change output with a replacing CPFP increasing package feerate to confirm soonly
+			- MUST replace according to BIP 125
+			- MUST reschedule timer for next bump
+		- MUST NOT spend more in bumping fees than its outcome value
+
+A node:
+	- if a CET transaction confirms:
+		- MUST note its outcome output for latter spend
+		- MUST advances its off-chain state to terminal phase
+
+A node:
+	- if `contract_timeout` is reached:
+		- MUST finalize the CET with a local `SIGHASH_ALL` signature
+		- MUST broadcast the refund transaction
+		- if the refund is not confirmed after 6 blocks:
+			- MUST spend the change output with a CPFP increasing package feerate to confirm soonly
+			- MUST start a bumping timer of length `congestion_delay`
+		- if `congestion_delay` expires:
+			- MUST spend the change output with a replacing CPFP increasing package feerate to confirm soonly
+			- MUST replace according to BIP 125
+			- MUST reschedule timer for next bump
+
+A node:
+	- if a CET transaction confirms:
+		- MUST note its outcome output for latter spend
+		- MUST advances its off-chain state to terminal phase
+
+
+# Fee bumping policy
+
+Client's `congestion_delay`/`security_delay` can be scheduled on three different basis:
+- a block height
+- a local clock
+- an observed feerate fluctuation
+
+We don't provide recommendations for the congestion delay as its settings depends of user
+liquidity preferences and available local data to predict the feerate.
+
+As `security_delay` must be offensive, we recommend to set it to 1 block.
```

Comment:
I think this section should be right after `General Requirements`, that way you have a better idea what these terms mean while reading.